cmd/main.go

```diff
@@ -117,7 +117,7 @@ func main() {
 		version.NewVersionCmd(),
 	)
 
-	foundProject, projectVersion := getProjectVersion()
+	foundProject, projectVersion := util.GetProjectVersion()
```

Comment:
Will we not add more than one thing in the "util". So, wdyt about it be utils? 
It is very strange to see util.

Reply:
I have no preference, just left the name that was already there.

@mengqiy @DirectXMan12 thoughts?

Reply:
Now that I give it a second though, this `util` package should only be accessed by the `cmd` package. What do you think about the special package `internal` or `internal/utils`?